src/Sarif.Viewer.VisualStudio.Core/SarifViewerPackage.cs

```diff
@@ -203,12 +215,60 @@ private void SolutionEvents_OnBeforeCloseSolution(object sender, EventArgs e)
         {
             // stop watcher when the solution is closed.
             this.sarifFolderMonitor?.StopWatch();
+
+            var fileSystem = new FileSystem();
+
+            try
+            {
+                fileSystem.FileDelete(Path.Combine(GetDotSarifDirectoryPath(), "scan-results.sarif"));
```

Comment:
>"scan-results.sarif" [](http://example.com/codeflow?start=79&length=20)

is this file name a default value that user can change, or is it fixed and always the same?

Reply:
This is not configurable. We may need more sophisticated cleanup code in the future but for now this is fine.